### PR TITLE
changelog

### PR DESCRIPTION
**Pull request by shironextcloud**
13.0.2

### CODE REVIEW

File: page-changelog.php [hunk 1]
```diff
@@ -12,6 +12,55 @@
...
+<li><a href="https://github.com/nextcloud/server/pull/9072">Fix proper permissions for multiple file access (server#9072)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/9073">When formatting a share node an Empty target is invalid (server#9073)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/9077">Move on with the next user if we found the user on one user back-end (server#9077)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/9089">Allow usage of Windows 10 WebDav Netdrive (server#9089)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/9094">Add more logging for the object storage during creation of the buckets (server#9094)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/9095">Do not convert email addresses with idn_to_ascii if… (server#9095)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/9124">Fix progress bar hidden before the upload ends (server#9124)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/9213">Update CRL to include old quicknotes cert (server#9213)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/9228">13.0.2 RC 1 (server#9228)</a></li>
```
Comment (MorrisJobke) [inline page-changelog.php]: This line could be removed (obviously we changed the version number for the new version) 😄

File: page-changelog.php [hunk 2]
```diff
@@ -171,6 +220,33 @@
...
+Many fixes were merged, the most important ones include:
+<ul>
+<li><a href="https://github.com/nextcloud/server/pull/8850">Group existence check works without attribute (like with users) (server#8850)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/8893">Fix undefined index problem (server#8893)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/8987">Show EOL warning in the update section (server#8987)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/9016">Update icewind/smb to 2.0.5 (server#9016)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/9078">Set new-password as autocomplete on the link password (server#9078)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/9088">Allow usage of Windows 10 WebDav Netdrive (server#9088)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/9214"> Update CRL to include old quicknotes cert (server#9214)</a></li>
+<li><a href="https://github.com/nextcloud/server/pull/9229">12.0.7 RC 1 (server#9229)</a></li>
```
Comment (MorrisJobke) [inline page-changelog.php]: Drop this as well and remove the unneeded spaces at the beginning of the text in the lines above and below this line.

File: strings.php [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 /* VERSIONS */
 $VERSIONS_SERVER_MAJOR_STABLE = '13';
-$VERSIONS_SERVER_FULL_STABLE = '13.0.1';
+$VERSIONS_SERVER_FULL_STABLE = '13.0.2';
```
Comment (MorrisJobke) [inline strings.php]: This should already do the trick with the server page AFAIK.

File: page-changelog.php (hunk 2 above)
Comment (shironextcloud) [reply page-changelog.php]: removed 
+<li><a href="https://github.com/nextcloud/server/pull/9229">12.0.7 RC 1 (server#9229)</a></li>

File: page-changelog.php (hunk 1 above)
Comment (shironextcloud) [reply page-changelog.php]: removed line for
+<li><a href="https://github.com/nextcloud/server/pull/9228">13.0.2 RC 1 (server#9228)</a></li>

**Comment by MorrisJobke** [comment]
* ✅ Uploaded to https://download.nextcloud.com/server/releases
* ✅ Update server tested

**Pull request merged**